packages/hosted-payment-integration/src/HostedPaymentMethod.tsx

```diff
@@ -16,12 +18,19 @@ const HostedPaymentMethod: FunctionComponent<PaymentMethodProps> = ({
     language,
     paymentForm,
 }) => {
+    const initializeHostedPaymentMethod = async (options: PaymentInitializeOptions) => {
+        return checkoutService.initializePayment({
+            ...options,
+            integrations: [createZipPaymentStrategy],
```

Comment:
If `zip` belongs here, could you please remove the zip payment in this list: https://github.com/bigcommerce/checkout-js/blob/c651d5ddbe9a22a38fa6a5c875014cb93c65e6ff/packages/core/src/app/payment/paymentMethod/PaymentMethod.tsx#L167-L176